care/abdm/api/viewsets/abha_number.py

```diff
@@ -28,7 +28,7 @@ def get_object(self):
             Q(abha_number=id) | Q(health_id=id) | Q(patient__external_id=id)
         ).first()
 
-        if not instance or get_patient_queryset(self.request.user).contains(
+        if not instance or not get_patient_queryset(self.request.user).contains(
```

Comment:
Can you add a relevant test case for this change?

Reply:
Currently, we don't have any tests for ABDM, opened up an issue (https://github.com/ohcnetwork/care_abdm/issues/12) to track it.